lib/matplotlib/tests/test_offsetbox.py

```diff
@@ -123,4 +125,58 @@ def test_get_packed_offsets(wd_list, total, sep, mode):
     # issue tickets (at least #10476 and #10784) related to corner cases
     # triggered inside this function when calling higher-level functions
     # (e.g. `Axes.legend`).
+    # These are just some additional smoke tests. The output is untested.
```

Comment:
It looks like this test could just be deleted as it is essentially covered by the tests below?

Reply:
Not exactly, e.g. I do not test negative `total` or `sep`. I agree that they'd better be tested explicitly for their output, but until then this is better than nothing.

Since it is not documented what *should* happen in this cases, I've left this as is (unresolved). IMHO that discussion is worth some thought and a separate PR.